src/polydatum/dal.py

```diff
@@ -84,13 +84,17 @@ class DataManager(object):
     """
     Registry for Services, Resources, and other DAL objects.
     """
+    DEFAULT_DAL_CLS = DataAccessLayer
 
     def __init__(self, resource_manager=None):
         if not resource_manager:
             resource_manager = ResourceManager(self)
 
         self._resource_manager = resource_manager
-        self._dal = DataAccessLayer(self)
```

Comment:
`self._dal = self.DataAccessLayer(self)`

Reply:
This seems so much simpler than what I suggested. 🤦‍♂️

Reply:
Class properties are default...by default.

Reply:
yea... but this attribute breaks instance member naming conventions in my opinion. I still think it's good though.

Reply:
It's a common pattern in the standard library and many open source projects. Since it references a class, class casing makes sense. For a pattern we normally use, this could be called `dal_factory`.